src/models/mentor.py

```
@@ -0,0 +1,585 @@
+                safe_org = quote(org, safe="")
+                safe_user = quote(username, safe="")
+                try:
+                    pr_resp = await github_api(
+                        "GET",
+                        f"/search/issues?q=is:pr+is:merged+org:{safe_org}+author:{safe_user}&per_page=1",
+                        token,
+                    )
+                    if pr_resp.status == 200:
+                        pr_data = json.loads(await pr_resp.text())
+                        merged_prs = int(pr_data.get("total_count") or 0)
+                except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)